workloads/network-perf/run_hostnetwork_network_test_fromgit.sh

```diff
@@ -2,29 +2,31 @@
 export WORKLOAD=hostnet
 
 source ./common.sh
+export HOSTNETWORK=true
 export pairs=1
 
-deploy_workload
-wait_for_benchmark
+run_workload ripsaw-uperf-crd.yaml
+if [[ $? != 0 ]]; then
+  exit 1
+fi
 assign_uuid
 run_benchmark_comparison
-print_uuid
 generate_csv
 
 if [[ ${ENABLE_SNAPPY_BACKUP} == "true" ]] ; then
- echo -e "snappy server as backup enabled"
- source ../../utils/snappy-move-results/common.sh
- csv_list=`find . -name "*.csv"` 
- mkdir files_list
- cp $csv_list ./files_list
- tar -zcvf snappy_files.tar.gz ./files_list
+  echo -e "snappy server as backup enabled"
+  source ../../utils/snappy-move-results/common.sh
+  csv_list=`find . -name "*.csv"` 
+  mkdir files_list
+  cp $csv_list ./files_list
+  tar -zcvf snappy_files.tar.gz ./files_list
+  
+  export workload=network_perf_hostnetwork_test
  
- export workload=network_perf_hostnetwork_test
-
- export snappy_path="$SNAPPY_USER_FOLDER/$runid$platform-$cluster_version-$network_type/$workload/$folder_date_time/"
- generate_metadata > metadata.json  
- ../../utils/snappy-move-results/run_snappy.sh snappy_files.tar.gz $snappy_path
- ../../utils/snappy-move-results/run_snappy.sh metadata.json $snappy_path
- store_on_elastic
- rm -rf files_list
-fi
\ No newline at end of file
+  export snappy_path="$SNAPPY_USER_FOLDER/$runid$platform-$cluster_version-$network_type/$workload/$folder_date_time/"
+  generate_metadata > metadata.json  
+  ../../utils/snappy-move-results/run_snappy.sh snappy_files.tar.gz $snappy_path
+  ../../utils/snappy-move-results/run_snappy.sh metadata.json $snappy_path
+  store_on_elastic
+  rm -rf files_list
+fi
```

Comment:
can we create a common function for ENABLE_SNAPPY_BACKUP in utils or common.sh so that all the workloads can use it